src/domain/LottoTicket.js

```diff
@@ -0,0 +1,28 @@
+import {
+  LOTTO_NUMBER_COUNT,
+  MAX_LOTTO_NUMBER,
+  MIN_LOTTO_NUMBER,
+} from "./constants.js";
+import { getRandomNumber } from "./utils.js";
+
+class LottoTicket {
+  #numbers = [];
+
+  constructor() {
+    this.#numbers = this.#createUniqueNumbers().sort((a, b) => a - b);
+  }
+
+  get numbers() {
+    return this.#numbers;
+  }
+
+  #createUniqueNumbers() {
+    const numbers = new Set();
+    while (numbers.size < LOTTO_NUMBER_COUNT) {
+      numbers.add(getRandomNumber(MIN_LOTTO_NUMBER, MAX_LOTTO_NUMBER));
+    }
+    return [...numbers];
+  }
```

Comment:
```suggestion
  #createUniqueNumbers() {
    return createUniqueNumbers(LOTTO_NUMBER_COUNT)
  }
```

이런 식으로 별도 유틸로 추출 후 로또 숫자의 개수를 인자로도 넣어줄 수 있을 것 같네요!

Reply:
https://github.com/next-step/js-lotto/pull/301/commits/52c97ca184b6695b9c5104a5396bb8377ad880b0